libsodium-sys/build.rs

```diff
@@ -88,6 +88,7 @@ fn main() {
             (format!("{}", gcc.get_compiler().path().display()),
              env::var("CFLAGS").unwrap_or(String::from(" -march=native -O3")))
         };
+        let cflags = format!("{} -fPIC", cflags);
         let prefix_arg = format!("--prefix={}", install_dir);
```

Comment:
I thought that `-fPIC` option can be applied only for a static library. But I'm not sure. Do you really sure this option  should be there ?